rotators/grbltrk/grbltrk.c

```diff
@@ -205,7 +205,7 @@ grbl_init(ROT *rot)
         if (retval != RIG_OK)
         {
             rot_debug(RIG_DEBUG_ERR, "grbl_request [%s] fail\n", grbl_init_list[i]);
-            return RIG_EIO;
```

Comment:
Return values are silently ignored by the caller. Looks like different bug (see ##1759).